tests/test_schema.py

```diff
@@ -0,0 +1,244 @@
+import types
+import pytest
+import inspect
+import datajoint as dj
+from unittest.mock import patch
+from inspect import getmembers
+from . import schema
+from . import PREFIX
+
+
+class Ephys(dj.Imported):
+    definition = """  # This is already declare in ./schema.py
+    """
+
+
+def relation_selector(attr):
+    try:
+        return issubclass(attr, dj.Table)
+    except TypeError:
+        return False
+
+
+def part_selector(attr):
+    try:
+        return issubclass(attr, dj.Part)
+    except TypeError:
+        return False
+
+
+@pytest.fixture
+def schema_empty_module(schema_any, schema_empty):
+    """
+    Mock the module tests_old.schema_empty.
+    The test `test_namespace_population` will check that the module contains all the
+    classes in schema_any, after running `spawn_missing_classes`.
+    """
+    namespace_dict = {
+        '_': schema_any,
+        'schema': schema_empty,
+        'Ephys': Ephys,
+    }
+    module = types.ModuleType('schema_empty')
+
+    # Add classes to the module's namespace
+    for k, v in namespace_dict.items():
+        setattr(module, k, v)
+
+    return module
```

Comment:
This was an interesting problem. In `tests_old/schema_empty.py`, `spawn_missing_classes` relies on the `schema.schema` (called the `schema_any` fixture in new tests) being activated on import. Because this is now in the `schema_any` fixture, we weren't able to create `schema_empty.schema` the same way in the pytest suite (in pytest, the value of the fixture is only available *after* everything's been imported). Instead, I used the `types.ModuleType` to mock a module that mimics `tests_old/schema_empty.py`.